File: app-components/src/main/java/org/synsystems/onlypass/components/preferences/GlobalPreferences.java
package org.synsystems.onlypass.components.preferences;

import org.synsystems.onlypass.framework.rxutils.Pulse;

import io.reactivex.Observable;

public interface GlobalPreferences {
  public Observable<Pulse> observeRemoteLoggingEnabled();

  public Observable<Pulse> observeRemoteLoggingDisabled();
}

File: app-components/src/main/java/org/synsystems/onlypass/components/preferences/GlobalPreferences.kt
package org.synsystems.onlypass.components.preferences

import io.reactivex.Observable
import org.synsystems.onlypass.framework.rxutils.Pulse

interface GlobalPreferences {
  fun observeRemoteLoggingEnabled(): Observable<Pulse>

  fun observeRemoteLoggingDisabled(): Observable<Pulse>
}